doc/tutorials/plugins.md

```diff
@@ -430,3 +433,10 @@ Plugin *ELEKTRA_PLUGIN_EXPORT(line)
 ```
 
 For further information see [the API documentation](https://doc.libelektra.org/api/current/html/group__plugin.html).
+
+## Note on Direct Method Calls via External Integrations
+
+Some applications want to call Elektra methods directly via native access.
```

Comment:
"Elektra methods" -> "functions exported by Elektra's plugin" otherwise the problem you describe will not occur.